ambry-network/src/main/java/com/github/ambry/network/http2/Http2BlockingChannel.java

```diff
@@ -74,16 +75,23 @@
   private static final Logger logger = LoggerFactory.getLogger(Http2BlockingChannel.class);
   private final static AttributeKey<Promise<ByteBuf>> RESPONSE_PROMISE = AttributeKey.newInstance("ResponsePromise");
   private final String hostName;
+  private final int readBufferSize;
+  private final int writeBufferSize;
+  private final int http2InitialWindowSize;
   private final SSLFactory sslFactory;
   private final int port;
   private EventLoopGroup workerGroup;
   private Channel channel;
   private Promise<ByteBuf> responsePromise;
   private Http2StreamChannelBootstrap http2StreamChannelBootstrap;
 
-  public Http2BlockingChannel(String hostName, int port, SSLConfig sslConfig) {
+  public Http2BlockingChannel(String hostName, int port, SSLConfig sslConfig, int readBufferSize, int writeBufferSize,
+      int http2InitialWindowSize) {
```

Comment:
Prefer to do it in this way.` BlockingChannel` is also doing the similar thing. It would be hard for test code refactor if `Http2BlokcingChannelConfig ` is introduced.